third_party/move/tools/move-resource-viewer/src/fat_type.rs

```diff
@@ -73,7 +73,7 @@ pub(crate) enum FatType {
     Address,
     Signer,
     Vector(Box<FatType>),
```

Comment:
We can probably make vectors pointers too? To reduce clones

Reply:
We could do that if the benchmark is not satisfactory, but I wouldn't expect too much from it. The real problem is the full transitive expansion of struct/enum. This is not it.